C_tasks-here/bentii/C_codes-task4.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+int main(){
+int sumOfNumber, firstNumber, secondNumber
+;
```

Comment:
correct this. it should be at the end of line 3. i.e. after secondNumber

Reply:
ok